src/common/utils/formValidation.ts

```diff
@@ -153,7 +153,7 @@ export const mustBeBusinessId = (value: string): string | undefined => {
 };
 
 export const mustBeBoatRegistrationNumber = (value: string): string | undefined => {
-  const regex = /^([PEL][0-9]{5,10})|([A-DF-KOQ-Z][0-9]{1,7})$/;
+  const regex = /^[A-ZÄÖÅ][1-9]{1,10}$/;
```

Comment:
This implementation works for and is correct based on the PR description. If you wanted to really allow all characters, you might be able to do that with `\p{L}`. Doesn't need to be done here, mentioning just as a curio

>In addition to complications, Unicode also brings new possibilities. One is that each Unicode character belongs to a certain category. You can match a single character belonging to the “letter” category with \p{L}. You can match a single character not belonging to that category with \P{L}.